org.buhocms.BuhoCMS.yml

```diff
@@ -0,0 +1,50 @@
+app-id: org.buhocms.BuhoCMS
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: buhocms
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+  - --filesystem=home
+modules:
+  - name: BuhoCMS
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/BuhoCMS
+      - cp -r buhocms data lib /app/BuhoCMS
+      - chmod +x /app/BuhoCMS/buhocms
+      - mkdir -p /app/bin
+      - ln -s /app/BuhoCMS/buhocms /app/bin/buhocms
+      - install -D installers/Flatpak/buhocms.svg -t /app/share/icons/hicolor/scalable/apps
+      - install -D installers/Flatpak/org.buhocms.BuhoCMS.desktop -t /app/share/applications
+      - install -D installers/Flatpak/org.buhocms.BuhoCMS.metainfo.xml -t /app/share/metainfo
+    sources:
+      - type: git
+        url: https://github.com/iakmds/buhocms.git
+        tag: v0.3.0
+        commit: 84d337921577a8cdca8dae17ced6e2967aa4596f
+      - type: archive
+        url: https://github.com/iakmds/buhocms/releases/download/v0.3.0/BuhoCMS-Linux-Portable.tar.gz
+        sha256: 3159db5103fc5b8056ab277dd3d56c1d9e50cb0b7d066686f75887e8a2664903
+        strip-components: 0
+  - name: hugo
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: https://github.com/gohugoio/hugo/releases/download/v0.111.1/hugo_0.111.1_linux-amd64.tar.gz
+        sha256: 22dd224edcbf214d3bfc9cf74b471cf475d8c15600119129bba30a57768a6b2d
+  - name: git
+    buildsystem: simple
+    make-args: INSTALL_SYMLINKS=1
+    make-install-args: INSTALL_SYMLINKS=1
```

Comment:
these are wrong too.

Reply:
I took it from the git-cola repo: https://github.com/flathub/com.github.git_cola.git-cola/blob/6afb53522e227f180dc3b57d418733002eaa6bac/git.json

Should I just remove them? Hopefully it installs git correctly